website/src/documentation/transforms/python/element-wise/tostring.md

```diff
@@ -19,9 +19,38 @@ limitations under the License.
 -->
 
 # ToString
+
+<script type="text/javascript">
+localStorage.setItem('language', 'language-py')
+</script>
+
 Transforms every element in an input collection a string.
 
-## Examples
-See [BEAM-7389](https://issues.apache.org/jira/browse/BEAM-7389) for updates. 
+## Example
+
+Any non-string element can be converted to a string using sandard Python functions and methods.
+Many I/O transforms such as `TextIO` expect their input elements to be strings.
+
+```py
+{% github_sample /apache/beam/blob/master/sdks/python/apache_beam/examples/snippets/transforms/element_wise/to_string.py tag:to_string %}```
+
+Output `PCollection` after *to string*:
```

Comment:
"The output `PCollection` from this pipeline is the following:"

Reply:
I just removed the "after to string" part. I'm hesitant to say that it's the output of the *pipeline* because the `print` function returns `None`, so the output to the pipeline is actually a `PCollection` of all `None`s. For testing we hack around that by mocking the print function to use `lambda element: element`.

Reply:
That makes more sense, thanks!